tests/iface_loopback_action/iface_loopback_action_helper.py

```diff
@@ -404,6 +411,29 @@ def remove_dut_portchannel_member(duthost, port, lag_port):
     """
     duthost.shell('config portchannel member del {} {}'.format(lag_port, port))
 
+    def _check_portchannel_member_removed(duthost, portchannel, member):
+        output = duthost.shell('show interface portchannel | grep "{} "'.format(portchannel))
+        if member + '(' not in output['stdout']:
+            return True
+        else:
+            return False
+    wait_until(10, 1, 0, _check_portchannel_member_removed, duthost, lag_port, port)
+
+
+def remove_dut_portchannel_ipv4_address(duthost, lag_port):
+    ipv4_address = re.split(r'\s+', duthost.shell('show ip interface | grep {}'.format(lag_port))['stdout'])[1]
+    if ipv4_address:
+        duthost.shell('sudo config interface ip remove {} {}'.format(lag_port, ipv4_address))
+
+
+def remove_dut_portchannel_ipv6_address(duthost, lag_port):
+    output = duthost.shell('show ipv6 interface | grep {}'.format(lag_port))['stdout_lines']
+    for line in output:
```

Comment:
1. Please use `-w` to do a full match
2. I don't know if the link local address (`fe80::x`) is always behind the IP address that we configured. So I think it's better to check if the IP address starts with `fe80`.

Reply:
Done